docs/getting-started-guides/mesos-docker.md

```diff
@@ -154,22 +161,38 @@ host machine (mac).
 
 1. Build binaries
 
+    You will need the server binaries compiled for linux/amd64 and kubectl compiled for your local architecture and operating system.
+
+    If you have Golang installed, you can build Kubernetes for your local architecture and operating system:
+
     ```
     KUBERNETES_CONTRIB=mesos hack/build-go.sh
     ```
 
-    Alternatively, you can use `make`, if make is installed.
+    If you have Golang and make installed, you can build Kubernetes for your local architecture and operating system with make:
+
+    ```
+    KUBERNETES_CONTRIB=mesos make
+    ```
 
-    Unless you're on linux, you'll also need to build the binaries for linux/amd64 (for the docker containers):
+    If you don't have or want Golang installed, you can build Kubernetes for linux/amd64 in a docker container:
```

Comment:
The primary reason is because I think people new to k8sm should have to install as few things as possible. Non-developers should be able to deploy k8sm without installing a programming language. CI's also should install as little as possible, and this documents the alternatives.